### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by benma** [comment]
fyi @jstrnbrg this adds the new PT translations.

File: frontends/web/src/locales/pt/app.json [hunk 1]
```diff
@@ -458,12 +569,24 @@
       "text": "O botão \"Enviar\" é ativado quando seu saldo for maior que zero.",
       "title": "Por que não consigo enviar {{unit}}?"
     },
+    "accountSummaryAmount": {
+      "link": {
+        "text": "www.coingecko.com",
+        "url": "https://www.coingecko.com/"
+      },
+      "text": "\nO montante total é a soma de de saldo de todas sua contas crpto. Taxas de troca são obtidas através da coingecko.com.\n\nObs: Se você usa a MyEtherWallet para tokens ela não é suportada no BitBoxApp, por isso não será incluso no montante total mostrado.",
```
Comment (thisconnect) [inline frontends/web/src/locales/pt/app.json:577]: I removed `\n` at the beginning in locize

**Review by thisconnect — APPROVED** [review]
LGTM with minor nit, already changed in locize

File: frontends/web/src/locales/pt/app.json (hunk 1 above)
Comment (benma) [reply frontends/web/src/locales/pt/app.json:577]: Merged before reading :see_no_evil: 

https://github.com/digitalbitbox/bitbox-wallet-app/pull/1398

I also fixed the `crpto` typo.

**Pull request merged**